library/src/doo/shell.clj

```diff
@@ -37,13 +37,19 @@
       (.close r)
       (.toString out))))
 
-(defn exec! [cmd]
+(defn exec* [base-dir command-arr]
+  (if base-dir
+    (.exec (Runtime/getRuntime) command-arr nil base-dir)
+    (.exec (Runtime/getRuntime) command-arr)))
+
+(defn exec! [cmd path]
```

Comment:
Could be good to make two arities for `exec!`, and have path be renamed to `working-dir`?